docs/reference/rest-api/logs.asciidoc

```diff
@@ -0,0 +1,278 @@
+
+The endpoint is designed in a way that logs are never dropped, as long as the cluster has enough capacity.
+
+If an error happens during ingestion,
+the logs are sent to the `logs-dlq-<namespace>` data stream that acts as a dead letter queue for failed events.
+However, log ingestion should rarely fail as the mappings in the built-in index template for the `logs-*-*` data streams are designed to minimize mapping conflicts.
+
+A <<logs-api-request-body, couple of fields>> from the {ecs-ref}[Elastic Common Schema (ECS)] are indexed by default that are commonly used to search or filter logs.
+All other fields are not indexed by default.
+But you can still add any top-level fields and use them in searches and aggregations as the default index template for logs
+<<dynamic-mapping-runtime-fields, maps dynamic fields as runtime fields>>.
+
+For custom fields that are frequently used in searches or aggregations, you might want to leverage the speed benefits of indexing the field.
+This comes at the expense of a larger index size more processing at ingest time.
+To index a custom field, <<create-index-template, create an index template>> for your dataset `logs-<dataset>-*` based on the built-in index template for `logs-*-*` and add your custom field to the mappings.
+To immediately apply the setting, <<manually-roll-over-a-data-stream, roll over the data stream>>
+This affects any new data added to the stream after the rollover.
+However, it does not affect the data stream’s existing backing indices or existing data.
+
+All fields, aside from the `@timestamp` field, are configured to <<ignore-malformed, ignore malformed>> values.
+This means that if a log event contains a field whose type is incompatible with the type of the field that exists in the mapping,
+{es} will ignore the field instead of rejecting the whole document.
+For example, when a string is provided for a field that is mapped to integer.
+Note that this currently doesn't apply for object/scalar mismatches, such as `"foo": "bar"` vs `"foo.bar": "baz"`.
+
+[discrete]
+[[logs-api-path-params]]
+=== {api-path-parms-title}
+
+{ecs-ref}/ecs-data_stream.html#field-data-stream-dataset[`data_stream.dataset`]::
+  (Optional, string)
+  Defaults to `generic`.
+  Describes the ingested data and its structure.
+  It is highly recommended to provide a value for this so that you can add structure to your logs after the fact.
+  Example: `nginx.access`.
+
+{ecs-ref}/ecs-data_stream.html#field-data-stream-namespace[`data_stream.namespace`]::
+  (Optional, string)
+  Defaults to `default`.
+  A user-configurable arbitrary grouping, such as an environment (dev, prod, or qa), a team, or a strategic business unit.
+
+[discrete]
+[[logs-api-query-params]]
+=== {api-query-parms-title}
+
+Any provided query parameter will be added to each log line.
+For example, `/_logs?service.name=myapp` will add `"service.name": "myapp"` to all logs.
```

Comment:
For later: I have the suspicion we will need the query params to set some other flags then adding data. Even though it looks very convenient, I wonder if this is the expected way of adding data. As we follow bulk requests, it does not fit into the body but it would likely be a better place for it?

Reply:
> I have the suspicion we will need the query params to set some other flags then adding data.

What I had in mind is that these flags would be prefixed by an underscore.